activities/activity_nick_grokhowsky.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+# Function to show significantly high and low pixels
+def run_highestPixelValues(elev, env, **kwargs):
+    gs.run_command(
+        "r.neighbors", input=elev, output="smooth", method="average", flags="c", env=env
+    )
+    # Calculate univariate statistics
+    stats = gs.parse_command("r.univar", map="smooth", flags="g", env=env)
+
+    # Parse the stats library and capture the values for mean and standard deviation
+    # for value in stats:
+    #    if 'mean' in value:
+    #        mean = float(value[ -7: ])
+    #    if 'standard deviation' in value:
+    #        sd = float(value[ -7: ])
+    # Calculate mean, standard deviation, and thresholds for high and low
+    mean = float(stats["mean"])
+    sd = float(stats["stddev"])
+    high = str(mean + (2 * sd))
+    low = str(mean - (2 * sd))
+    # Using map algebra create a new raster map of highest and lowest pixel values versus all others
+    gs.mapcalc("high_values=if(smooth >= " + high + ")", env=env)
+    gs.mapcalc("new_values=high_values + 1", env=env)
+    gs.mapcalc("low_values=if(smooth > " + low + ")", env=env)
+    gs.mapcalc("final_values=high_values + low_values", env=env)
```

Comment:
Please rewrite this, it is unclear and new_values are not even used.

Reply:
I fixed this in the last push that was accepted.  It is a little strange that the overwrite flag is a separate parameter where other flags are passed by the flag parameter.  Fortunately, I figured it out and I was able to get this done with just one raster image, and I decided to keep all insignificant values as their original pixel value.  Also, I stayed with significance outside of 68%  (1 sd) because there was not much to show with larger magnitudes of sd.